cluster-autoscaler/processors/nodes/types.go

```diff
@@ -42,3 +42,12 @@ type ScaleDownSetProcessor interface {
 	// CleanUp is called at CA termination
 	CleanUp()
 }
+
+type ScaleDownDelayStatus struct {
+	ScaleDownDelayAfterAdd ScaleDownDelay
+}
+
+type ScaleDownDelay struct {
+	PerNG      map[string]bool
+	Aggregated bool
```

Comment:
What is the purpose of this field? You don't seem to be using it anywhere.

Reply:
Thanks for the comment. This might go away. I had initially added it with an approach which might not make as much sense anymore.

Reply:
Removed `Aggregated` field.